packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -680,6 +693,21 @@ export const Webchat = forwardRef((props, ref) => {
       </div>
     )
   }
+  const [chatAreaHeight, setChatAreaHeight] = useState(0)
+  useEffect(() => {
+    const webchatHeight = document.getElementById(
+      BotonicContainerId.Webchat
+    )?.clientHeight
+    const headerHeight = document.getElementById(
+      BotonicContainerId.Header
+    )?.clientHeight
+    const inputPanelHeight = document.getElementById(
+      BotonicContainerId.InputPanel
+    )?.clientHeight
+    if (webchatHeight && headerHeight && inputPanelHeight) {
+      setChatAreaHeight(webchatHeight - headerHeight - inputPanelHeight)
+    }
+  }, [webchatState.isWebchatOpen])
```

Comment:
We should encapsulate this logic in a custom hook

Reply:
yep, done in the next PR